context/app/static/js/pages/Sample/Sample.jsx

```diff
@@ -35,15 +37,49 @@ function SampleDetail(props) {
     metadata,
     rui_location,
   } = assayMetadata;
+  const { elasticsearchEndpoint, nexusToken } = useContext(AppContext);
+
+  const { searchData: sampleSpecificDatasets } = useSearchHits(
+    JSON.stringify({
```

Comment:
Just trying to contribute, is there an advantage to maybe memoizing this first instead of `stringify` on every call?  I think `stringify` can be a little price-y in terms of performance but maybe I am splitting hairs/projecting my own concerns

Reply:
Also memo-izing can also be expensive so maybe not...

Reply:
I ended up just using `useMemo` in `Sample` and pushing `stringify` inside the hook.